syftlib/src/main/java/org/openmined/syft/Syft.kt

```diff
@@ -1,52 +1,38 @@
 package org.openmined.syft
 
 import io.reactivex.disposables.CompositeDisposable
-import kotlinx.serialization.ImplicitReflectionSerializer
-import kotlinx.serialization.json.JsonObject
-import kotlinx.serialization.json.JsonPrimitive
-import org.openmined.syft.domain.Protocol
-import org.openmined.syft.network.NetworkMessage
+import org.openmined.syft.network.CommunicationDataFactory
+import org.openmined.syft.network.REQUEST
 import org.openmined.syft.network.SignallingClient
 import org.openmined.syft.threading.ProcessSchedulers
 
 @ExperimentalUnsignedTypes
-class Syft constructor(
+class Syft private constructor(
     private val signallingClient: SignallingClient,
     private val schedulers: ProcessSchedulers
 ) {
+    private lateinit var syftWorker: Syft
 
-    private val compositeDisposable = CompositeDisposable()
-
-    @ImplicitReflectionSerializer
-    fun start() {
-
-        val disposable = signallingClient.start()
-                .map {
-                    when (it) {
-                        is NetworkMessage.SocketOpen -> {
-                            println("Socket open")
-                            send("And now that we are opened, I send a message")
-                        }
-                        is NetworkMessage.SocketClosed -> println("Socket was closed successfully")
-                        is NetworkMessage.SocketError -> println(it.throwable.message)
-                        is NetworkMessage.MessageReceived -> println(it)
-                        is NetworkMessage.MessageSent -> println("Message sent successfully")
-                    }
-                }
+    private val workerJobs = mutableListOf<Job>()
```

Comment:
Maybe a `map` so we can know if a worker has already being created here.

Reply:
I just removed it for now. Essentially, we will need to implement all the functionality here. I am kinda focussing on jobs atm so left it for now